src/mode/text.js

```diff
@@ -57,8 +51,14 @@ var Mode = function() {
         if (!this.lineCommentStart) {
             if (!this.blockComment)
                 return false;
+            /**
+             * @type {any}
+             */
```

Comment:
When you are done, I hope you have no any's left :smirk: 

You can also use `/** @type {any} */` to not waste vertical space.

Reply:
@mkslanc i agree, putting this on one line is better